task8/src/main/java/oldServlets/Check.java

```diff
@@ -0,0 +1,30 @@
+package com.example.demo;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.io.PrintWriter;
+
+@WebServlet(name = "check", value = "/check")
+public class Check extends HttpServlet {
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException, ServletException {
+        doPost(request, response);
```

Comment:
Please never switch the type of request secretly from a user. I understand this is just a "dummy" sample, but still this is outright wrong)